lib/src/models/token.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'token.freezed.dart';
+
+@freezed
+sealed class Token with _$Token {
+  Token._();
+
+  factory Token({
```

Comment:
This file is prob not needed? `ParsedToken` is already in use

Reply:
done